_data/projects/Please.yml

```diff
@@ -1,14 +0,0 @@
-name: Please.com
-desc: The Open Source Trip Planner and Booking Service
-site: https://github.com/pleasedotcom
-tags:
-# Note: these are tags categorizing the project, not issue labels.
-- javascript
-- react
-- web
-- html
-- serverless
-- node
-upforgrabs:
-  name: good first issue
-  link: https://github.com/PleaseDotCom/frontend/labels/good%20first%20issue
```

Comment:
@ramsanath I gather this is the project referred to here:

> Removed a discontinued project.

I checked the link here and see there are 3 open issues with this label, which makes me think it's still open for new . contributors. Can you elaborate on why you think this project can be removed?

Reply:
Hi, Please refer to the README of that repository

> Due to an unsuccessful ICO, Please has ceased to exist. Hower the project has been taken over by Deens

Most of the issues there are a year old and there is no source code in that repository.
I had planned to add the new repository 'Deens' in a separate branch.

Reply:
Ah, that's the context I was missing! Thanks @ramsanath!